Source/WebKit/WebProcess/WebPage/WebFrame.cpp

```diff
@@ -877,20 +886,27 @@ JSGlobalContextRef WebFrame::jsContextForWorld(DOMWrapperWorld& world)
 
 JSGlobalContextRef WebFrame::jsContextForWorld(InjectedBundleScriptWorld* world)
 {
-    return jsContextForWorld(world->coreWorld());
+    RefPtr localWorld = world;
```

Comment:
Even for other things than frame, I don't like the `local` prefix and this is not common practice in our codebase.